File: Laboratorios C++/TarjetaBiblioteca/TarjetaBiblioteca.cpp
#include <iostream>
using namespace std;

int main(){
    string tarjeta;  //guardar la palabra

    cout << "Inserte su tarjeta de biblioteca y dé el estado de ella (activa/inactiva): "; //ingresar si la tarjeta está activa o no
    cin >> tarjeta;

    //verifica la palabra ingresada
    if(tarjeta == "activa"){
        cout << "El usuario puede hacer préstamos." <<endl;
    }

    else{
        cout << "El usuario no puede hacer préstamos." <<endl;
    }
    return 0;
}

File: Laboratorios Java/TarjetaBiblioteca/src/TarjetaBiblioteca.java
import java.util.Scanner;
public class TarjetaBiblioteca{
    public static void main(String[] args) throws Exception {
        Scanner scanner = new Scanner(System.in);

        //Solicitar al usuario que ingrese el estado de la tarjeta
        System.out.print("Ingrese el estado de su tarjeta de biblioteca (1 para activo y 2 para inactivo): ");
        int estadoTarjeta = scanner.nextInt();

        //verificar si la tarjeta está activa o inactiva
        if(estadoTarjeta == 1){
            //tarjeta activa
            System.out.println("Su tarjeta está activa. Usted puede realizar préstamos.");
        }

        if(estadoTarjeta == 2){
            //tarjeta inactiva
            System.out.println("Su tarjeta está inactiva. No se permiten préstamos.");
        }
        scanner.close();
    }
}